src/current/_includes/cockroachcloud/backups/terraform-managed-backups.md

```diff
@@ -0,0 +1,9 @@
+Set the following for the `backup_config` attribute:
+
+- `enabled` controls whether managed backups are enabled or disabled. If you are disabling managed backups, you cannot set backup frequency or retention. Possible values are: `true` or `false`.
```

Comment:
> If you are disabling managed backups, you cannot set backup frequency or retention.

These values can still be set when managed backups are disabled.  Of course backups will be disabled so they won't take affect.  But setting retention even while backups are disabled uses up your one edit IIRC.

Reply:
I removed the line I had and just added that modifying `retention_days` even with backups disabled would use the one change.